lib/hyrax/controlled_vocabulary/importer/language.rb

```diff
@@ -23,7 +23,7 @@ def import
             'languages',
             [rdf_path],
             format: 'rdfxml',
-            predicate: RDF::Vocab::SKOS.prefLabel
+            predicate: 'http://www.w3.org/2008/05/skos#prefLabel'
```

Comment:
It works either way so if you'd rather I use RDF::URI('http://www.w3.org/2008/05/skos#prefLabel') I'll happily change it.